### DISCUSSION TITLE
How Large Can Man Mesh Grid Size Be Set?

### DISCUSSION THREAD

**Discussion by Cal-LF** [Jyers/Marlin#897] [Q&A] [answered]
During compiling the 3X3 Man Mesh I tried a 4X4 grid. I like this better than either the 3 or 5 grid. What is the limit for manual mesh grid size?

**Comment by AktasC** [comment, marked as answer]
AFAIK you could literally take his source code and change it to 10 by 10 if you wanted to, by just changing two values
